sound/soc/sof/intel/hda-dai.c

```diff
@@ -414,6 +414,16 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 #endif
 #endif
 
+#define SND_SOC_PLAYBACK_STREAM(_name, _min, _max) .playback = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,},
+
+#define SND_SOC_CAPTURE_STREAM(_name, _min, _max) .capture = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,},
```

Comment:
please, don't include the terminating comma in the macro definition. In this case it doesn't cause any problems because you never have more than one of these macros used, but in general you don't want struct definitions like
```
struct x xx = {
	X(1)
	X(2)
	X(3)
};
```
instead you want
```
struct x xx = {
	X(1),
	X(2),
	X(3),
};
```
So, in your case please do
```
+#define SND_SOC_PLAYBACK_STREAM(_name, _min, _max) .playback = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,}
+
+#define SND_SOC_CAPTURE_STREAM(_name, _min, _max) .capture = \
+	{.stream_name = _name, .channels_min = _min, .channels_max = _max,}
```

Reply:
Thanks @lyakh , update it now